index.html

```diff
@@ -124,14 +138,107 @@ <h1>Multi-Post Stories</h1>
+            <li><a href="#" class="btn">CSS</a></li>
+            <li><a href="#" class="btn">javascript</a></li>
+          </ul>
+          <button class="last-btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="container-bc">
+        <div class="content">
+          <h1>Multi-Post Stories</h1>
+          <ul>
+            <li>FACEBOOK</li>
+            <li>Full Stack Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            Experimental content creation feature that allows users to add to an existing story over the course of a day
+            without spamming their friends.
+          </p>
+          <ul class="btn-div">
+            <li><a href="#" class="btn">HTML</a></li>
+            <li><a href="#" class="btn">CSS</a></li>
+            <li><a href="#" class="btn">Ruby on rails</a></li>
+            <li><a href="#" class="btn">javascript</a></li>
+          </ul>
+          <button class="last-btn" type="button">See Project</button>
+        </div>
+        <img src="pimages/Snapshoot Portfolio (3).png" />
+      </div>
+      <div class="container-bc">
+        <div>
+          <img src="pimages/Snapshoot Portfolio.png" />
+        </div>
+        <div class="content">
+          <h1>Facebook 360</h1>
+          <ul>
+            <li>FACEBOOK</li>
+            <li>Full Stack Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            Experimental content creation feature that allows users to add to an existing story over the course of a day
+            without spamming their friends.
+          </p>
+          <ul class="btn-div">
+            <li><a href="#" class="btn">HTML</a></li>
+            <li><a href="#" class="btn">CSS</a></li>
+            <li><a href="#" class="btn">Ruby on rails</a></li>
+            <li><a href="#" class="btn">javascript</a></li>
+          </ul>
+          <button class="last-btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="container-bc">
+        <div class="content">
+          <h1>Uber Navigation</h1>
+          <ul>
+            <li>UBER</li>
+            <li>Lead Deveveloper</li>
+            <li>2018</li>
+          </ul>
+          <p>
+            A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer:
+            your car.
+          </p>
+          <ul class="btn-div">
+            <li><a href="#" class="btn">HTML</a></li>
+            <li><a href="#" class="btn">CSS</a></li>
+            <li><a href="#" class="btn">Ruby on rails</a></li>
+            <li><a href="#" class="btn">javascript</a></li>
+          </ul>
+          <button class="last-btn" type="button">See Project</button>
+        </div>
+        <img src="pimages/Snapshoot Portfolio (1).png" />
+      </div>
+    </section>
     <section class="section-c">
       <div class="container-c">
         <div class="about-div">
```

Comment:
- [ ] As mentioned by the previous reviewer to stick as much as possible to Figma design, I think there are things should improve again such as the `fonts(size, weight,...)`, `color` on some text, `spacing`(padding or margin), `alignment`,...
Just take a look on the difference on the above 2 images:
### Figma Design 
![Screenshot from 2023-05-12 17-57-57](https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/83042742/e1cb4998-fca7-4fd6-a432-81f1e73281e7)

### Your Design 
![Screenshot from 2023-05-12 18-17-39](https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/83042742/09977bf7-74fa-4c98-bfd3-00988d4dabc9)